### PR TITLE
[record-minmax-conversion-test] Revise test

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit revises the test to reduce test duration.

Related: #12071

### Before
```bash
Start 56: record_minmax_conversion_test
56/76 Test #56: record_minmax_conversion_test ..........   Passed   18.16 sec
```

### After
```bash
Test project /home/seongwoo/ONE/build
    Start 57: record_minmax_conversion_test
1/1 Test #57: record_minmax_conversion_test ....   Passed    1.52 sec

100% tests passed, 0 tests failed out of 1

Total Test time (real) =   1.53 sec
```
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

**Comment by mhs4670go** [comment]
To reduce the number of tensorflow importing, I revised to generate inputs at once.

File: compiler/record-minmax-conversion-test/testall.sh [hunk 1]
```diff
@@ -25,6 +25,20 @@ PASSED=()
...
+source "${VIRTUALENV}/bin/activate"
+  "${VIRTUALENV}/bin/python" "${GEN_SCRIPT_PATH}" \
+  --num_data 3 \
+  --output_dir "${BIN_PATH}" \
+  --artifact_dir ${WORKDIR} \
+  --model "$@"
+
+if [[ $? -ne 0 ]]; then
+  echo "FAILED TO GENERATE INPUT"
+  continue
```
Comment (jinevening) [inline compiler/record-minmax-conversion-test/testall.sh:39]: This is not in the loop anymore.

```suggestion
  exit 255
```

File: compiler/record-minmax-conversion-test/gen_h5_random_inputs_all.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+import h5py as h5
+import numpy as np
+import tensorflow as tf
+import argparse
+import os
+
+#
+# This script generates a pack of random input data (.h5) expected by the input tflite model
```
Comment (jinevening) [inline compiler/record-minmax-conversion-test/gen_h5_random_inputs_all.py:9]:
```suggestion
# This script generates a pack of random input data (.h5) expected by the input tflite models
```

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**